### PR TITLE
Created "Works" and "About me" sections

### PR DESCRIPTION
**Pull request by toliboff**
Created "Works" and "About me" sections, added styles  for mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,5 +37,131 @@ <h1>
...
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
```
Comment (descholar-ceo) [inline index.html]: - [ ] The list of projects should use a ul tag to group them, and li per project

File: index.html [hunk 2]
```diff
@@ -38,5 +37,131 @@ <h1>
...
+
+    <section class="about-me">
+        <h1>
+           About Myself
+        </h1>
+
+        <p>
+            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a href="#" class="connect-me">LET'S CONNECT</a>
```
Comment (descholar-ceo) [inline index.html]: - [ ] Since this `Let's connect` is referring to the following social media icons, then it should be header using `hx` tag

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @toliboff, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: It's nice to see that you worked on the requested change. good job!

File: index.html (hunk 2 above)
Comment (DeliceLydia) [reply index.html]: Well resolved the issue!

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved ✨✨🎉


Congratulations @toliboff ! 🎉 for making all the changes requested by the previous TSE and I can see that you went the extra mile to what was required, it is always inspiring to see hardworking people it bring a smile to my face! Your project is complete!  There is nothing else to say other than... it's time to merge it and move on to the next project.


### Keep Rocking!👏👏💪

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**